components/life-cycle-extension/src/main/java/com/wso2telco/dep/lifecycleextension/notification/impl/NotificationServiceImpl.java

```diff
@@ -53,35 +53,39 @@ public void sendApiProviderEmail(RequestContext context) {
 
         try {
 
-        GenericArtifactManager artifactManager = APIUtil.getArtifactManager(context.getSystemRegistry(),
+            log.info("Starting email trigger in Api creation ");
```

Comment:
change the log message to 

`"sending API creation email to all users in " + Constants.ADMIN_ROLE + " role"`